recipes/openvino/all/conanfile.py

```diff
@@ -301,14 +313,18 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
+        debug = "d" if self.settings.build_type == "Debug" and self.settings.os == "Windows" else ""
+
         self.cpp_info.set_property("cmake_find_mode", "config")
         self.cpp_info.set_property("cmake_file_name", "OpenVINO")
         self.cpp_info.set_property("pkg_config_name", "openvino")
 
         openvino_runtime = self.cpp_info.components["Runtime"]
         openvino_runtime.set_property("cmake_target_name", "openvino::runtime")
-        openvino_runtime.requires = ["onetbb::libtbb", "pugixml::pugixml"]
-        openvino_runtime.libs = ["openvino"]
+        openvino_runtime.requires = ["pugixml::pugixml"]
+        if self.options.threading == "tbb":
+            openvino_runtime.append("onetbb::libtbb")
+        openvino_runtime.libs = [f"openvino{debug}"]
```

Comment:
maybe we can create a function `def l()` which will return proper name? In this case all libs are wrapped with `l('openvino')` which is less verbose